src/main/java/kitchenpos/products/tobe/domain/model/Price.java

```diff
@@ -0,0 +1,37 @@
+package kitchenpos.products.tobe.domain.model;
+
+import java.math.BigDecimal;
+import javax.persistence.Embeddable;
+
+@Embeddable
+public class Price {
+
+    private static final BigDecimal PRICE_TRESHHOLD = BigDecimal.ZERO;
+    private BigDecimal price = BigDecimal.ZERO;
+
+    public Price() {
+    }
+
+    public Price(BigDecimal price) {
+        validatePrice(price);
+        this.price = price;
+    }
+
+    public BigDecimal getPrice() {
+        return price;
+    }
+
+    private void setPrice(BigDecimal price) {
+        this.price = price;
+    }
+
+    private void validatePrice(BigDecimal price) {
+        if (price == null) {
```

Comment:
```suggestion
        if (Objects.isNull(price)) {
```
Objects를 활용해보는건 어떨까요~^^